bundle/manifests/splunk-operator.clusterserviceversion.yaml

```diff
@@ -488,6 +488,7 @@ spec:
                   allowPrivilegeEscalation: false
               securityContext:
                 runAsUser: 1001
+                fsGroup: 1001
```

Comment:
Hi @sgontla , when does this file come into the picture? Isn't [manager.yml](https://github.com/splunk/splunk-operator/blob/feature-appframework/config/manager/manager.yaml) used to generate the splunk-operator-install.yaml? :)

Reply:
you are right. I think I checked out the wrong file before the commit :-). I couldn't test myself due to the docker issue.